src/scmdata/run.py

```diff
@@ -1694,7 +1694,7 @@ def process_over(
             ts.index = pd.MultiIndex.from_frame(ts_idx.fillna(na_override))
 
         group_cols = list(set(ts.index.names) - set(cols))
-        grouper = ts.groupby(group_cols)
+        grouper = ts.groupby(group_cols, group_keys=False)
```

Comment:
This removes a warning for pandas>1.5 and conserves the default behaviour from v1.4. See https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.groupby.html

Reply:
Nice. I just made #225 which is related I think